decidim-comments/app/cells/decidim/comments/comment_cell.rb

```diff
@@ -243,6 +243,10 @@ def vote_button_to(path, params, &)
 
         action_authorized_button_to(:vote_comment, path, params.merge(resource: root_commentable), &)
       end
+
+      def decidim_verifications
```

Comment:
I thought the verifications module is required by the decidim core since the `Decidim::Authorization` model is associated with verifications. The functionality of this PR also depends on this verifications flow but I can investigate a way to disable it if the module is not installed.

Reply:
forget it for now